compatibility/versions/UpdateVersions.hs

```diff
@@ -168,15 +162,16 @@ optsParser :: Parser Opts
 optsParser = Opts
   <$> strOption (short 'o' <> help "Path to output file")
 
+getVersionsFromTags :: IO (Set Version, Set Version)
+getVersionsFromTags = do
+    tags <- lines <$> System.Process.readProcess "git" ["tag"] ""
+    let versions = Set.fromList $ rights $ map (SemVer.fromText . T.pack . tail) tags
```

Comment:
```suggestion
    let versions = Set.fromList $ rights $ mapMaybe (fmap (SemVer.fromText . T.pack) . stripPrefix "p") tags
```
A bit more explicit. I’m also fine with just a comment as to why we call `tail`. Note that `tail` crashes on an empty list (not that this matters here).

Reply:
I imagine you meant `stripPrefix "v"`?

Reply:
Eh yeah, sorry about that.

Reply:
Made the change.